addons/web/static/src/core/assets.js

```diff
@@ -60,7 +71,7 @@ const loadJS = memoize(function loadJS(url) {
     scriptEl.src = url;
     document.head.appendChild(scriptEl);
     return new Promise((resolve, reject) => {
-        scriptEl.addEventListener("load", resolve);
+        scriptEl.addEventListener("load", () => resolve(url));
```

Comment:
I resolve the promise with the URL that has been used so it can be added as a value to bundlesCache

Reply:
I don't think we need this: there is nothing useful the caller can do with this URL, and the loadJS and loadCSS functions are already memoized. The only plausible thing the caller can do with these URLs is try to load them themselves by hand and end up double loading them, so it's a needless footgun in my opinion.